stl/inc/complex

```diff
@@ -186,10 +200,22 @@ public:
         return (numeric_limits<long double>::max)();
     }
 
+    static _Ty _Abs(_Ty _Left) {
+        // this always returns the correct value
+        // _Left < 0 ? -_Left : _Left is incorrect when _Left is -0.0

```

Comment:
I'm not a fan of "this other thing that isn't written here would be wrong" comments - the set of incorrect implementations is unbounded. That said, these are less egregiously offensive than "this is correct" and I could live with them.

Reply:
What about something like `// testing _Left < 0 would be incorrect`? We could also cite this PR or the linked issue.

Reply:
I'd prefer tests to comments: a test failure is a much stronger barrier to regression than a comment. `// testing _Left < 0 would be incorrect` _would_ be an improvement, however, and I don't think it's worth burning hours to reverse engineer test cases for every caller of these functions.

Reply:
Changed comments to `// testing _Left < 0 would be incorrect when _Left is -0.0`